src/EVM/Solvers.hs

```diff
@@ -112,21 +112,31 @@ withSolvers solver count timeout cont = do
       _ <- forkIO $ runTask task inst avail
       orchestrate queue avail
 
-    runTask (Task (SMT2 cmds cexvars) r) inst availableInstances = do
+    runTask (Task (SMT2 cmds (AbstData abstData) cexvars) r) inst availableInstances = do
       -- reset solver and send all lines of provided script
-      out <- sendScript inst (SMT2 ("(reset)" : cmds) cexvars)
+      out <- sendScript inst (SMT2 ("(reset)" : cmds) undefined undefined)
```

Comment:
So I replaced these with mempty. I want to make it clear when reading the code that `sendScript` doesn't actually do anything with the other elements of SMT2. This is kinda important, because we `sendScript` twice, once with the commands, and once with the `refinementEqs`.

I hope the mempy is good!